SECURITY.md

```diff
@@ -0,0 +1,19 @@
+# Security Policy
+
+## Supported Versions
+
+Only the latest version is supported.
+
+## Reporting a Vulnerability
+
+If you believe you have found a security vulnerability in SciCat, please report it to us through coordinated disclosure.
+
+**Please do not report security vulnerabilities through public GitHub issues, discussions, or pull requests.**
+
+Instead, please email <ADDRESS>
+
+Please include as much information as you can to help us better understand and resolve the issue.
+
+## Disclosure
+
+We use GitHub [security advisories](https://github.com/SciCatProject/scitacean/security/advisories) to disclose fixed vulnerabilities.
```

Comment:
After change the URL to https://github.com/SciCatProject/scicat-backend-next/security/advisories, we should get it merged

Reply:
Updated the link